gensim/corpora/svmlightcorpus.py

```diff
@@ -98,7 +99,7 @@ def save_corpus(fname, corpus, id2word=None, labels=False, metadata=False):
             Corpus in BoW format.
         id2word : dict of (str, str), optional
             Mapping id -> word.
-        labels : list or False
+        labels : iterable of ELEMENT_TYPE or False
```

Comment:
ELEMENT_TYPE is a placeholder: please replace by the actual type that is expected inside `labels`.